src/StoreKeeper/WooCommerce/B2C/Models/TaskModel.php

```diff
@@ -188,7 +195,7 @@ public static function getExecutionDurationSumByCreatedDateTimeRange($start, $en
     {
         global $wpdb;
 
-        $select = static::prepareCreatedDateTimeRangeSelect($start, $end);
+        $select = static::prepareProcessedDateTimeRangeSelect($start, $end);
```

Comment:
the name `public static function getExecutionDurationSumByCreatedDateTimeRange($start, $end): ?float` is not correct anymore

Reply:
Changed name of method to `getExecutionDurationSumByProcessedDateTimeRange`.